Dockerfile

```diff
@@ -0,0 +1,20 @@
+
+
+FROM gcr.io/cloud-builders/gcloud
+
+# Copy everything into the container
+COPY . /
+RUN ls
+
+RUN apt-get -y update
+RUN apt-get -y install python3-pip
+
+ENV PATH=$PATH:/builder/google-cloud-sdk/bin/
+ENV PYTHONPATH /
+
+RUN pip install -r requirements.txt
```

Comment:
Rename `requirements.txt` to `requirements.pip`?

Reply:
Changed the extension.